### PR TITLE
Create semreg-cli.md

### PR DESCRIPTION
**Pull request by bogyo210**

### CODE REVIEW

**Review by mateuscap — APPROVED** [review]

File: semreg-cli.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# semreg-cli
+To use this CLI in your jobs you need to do the following
+
+### Step 1 - install the CLI
+Installation :
```
Comment (csidyel) [inline semreg-cli.md:5]: To download and install sem, copy and paste the installation command:

File: semreg-cli.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# semreg-cli
```
Comment (csidyel) [inline semreg-cli.md:1]: Here there should go the description:

---
description: ...
---

File: semreg-cli.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+To use this CLI in your jobs you need to do the following
```
Comment (csidyel) [inline semreg-cli.md:2]: Here should go the purpose of the semreg-cli.

File: semreg-cli.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+### Step 1 - install the CLI
+Installation :
+
+`wget https://github.com/renderedtext/snippets/blob/master/semreg -O semreg && source semreg`
```
Comment (csidyel) [inline semreg-cli.md:7]:
``` bash
wget https://github.com/renderedtext/snippets/blob/master/semreg -O semreg && source semreg
```

**Review by csidyel — APPROVED** [review]

**Review by d-stefanovic — APPROVED** [review]

**Pull request merged**